client.go

```diff
@@ -22,20 +22,32 @@ const (
 type sqsOp int
 
 const (
-	opSend   sqsOp = iota
-	opDelete sqsOp = iota
+	opSend             sqsOp = iota
+	opDelete           sqsOp = iota
+	opChangeVisibility sqsOp = iota
 )
 
 // SQSClient wraps *sqs.Client from aws-sdk-go-v2
 type SQSClient interface {
-	SendMessageBatch(context.Context, *sqs.SendMessageBatchInput, ...func(*sqs.Options)) (*sqs.SendMessageBatchOutput, error)
-	DeleteMessageBatch(context.Context, *sqs.DeleteMessageBatchInput, ...func(*sqs.Options)) (*sqs.DeleteMessageBatchOutput, error)
+	SendMessageBatch(context.Context,
+		*sqs.SendMessageBatchInput,
+		...func(*sqs.Options)) (*sqs.SendMessageBatchOutput, error)
+	DeleteMessageBatch(context.Context,
+		*sqs.DeleteMessageBatchInput,
+		...func(*sqs.Options)) (*sqs.DeleteMessageBatchOutput, error)
+	ReceiveMessage(context.Context,
```

Comment:
I don't see a point of having `ReceiveMessage` as part of this lib since SQS allows you to fetch upto 10 messages in a single batch as a blocking operation.

Reply:
yeah, but added few things like visiblity timeout and all so all the required function can be found at single place instead of using two different object for the same.